glossary.md

```diff
@@ -42,6 +35,6 @@ USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
 
-- **Meta Content Framework (MCF)**
+- **DCID**
```

Comment:
I think it would be good to keep this alphabetically sorted (or we should at least introduce DCID first)

Reply:
This is a great point, thanks for this!